python/ray/cluster_utils.py

```diff
@@ -16,7 +16,7 @@
 
 logger = logging.getLogger(__name__)
 
-cluster_not_supported = os.name == "nt"
+cluster_not_supported = False
```

Comment:
Will remove this change as well. Instead I will ignore `cluster_not_supported` in the pytest fixture, `ray_cluster_start`.